tests/unit/button/button.html

```diff
@@ -74,6 +77,7 @@
 <div><input id="submit" type="submit" value="Label"></div>
 
 <button id="button1">Button</button>
```

Comment:
Awesome naming. While we're at it, let's drop this element and use #button instead? Now that you've removed the `class` (above), it should work just fine.